src/cirrus/management/commands/manage.py

```diff
@@ -343,3 +302,57 @@ def list_lambdas(ctx, deployment: Deployment, session: Session):
             default=str,
         ),
     )
+
+
+@manage.command("get-payloads")
+@query_filters
+@pass_deployment
+def get_payloads(
+    deployment: Deployment,
+    collections_workflow: str,
+    state: str | None,
+    since: str | None,
+    error_prefix: str | None,
+    limit: int | None,
+):
+    """
+    Retrieve a filtered set of payloads from S3 via querying the state DB for
+    matching payload IDs, altering paylods to enable rerun.
+    """
+    os.environ["CIRRUS_STATE_DB"] = deployment.environment["CIRRUS_STATE_DB"]
+    statedb = StateDB()
+
+    query_args = {
+        "state": state,
+        "since": since,
+        "error_begins_with": error_prefix,
+    }
+
+    items = statedb.get_items(
+        collections_workflow=collections_workflow,
+        limit=limit,
+        **query_args,
+    )
+    for item in items:
+        with BytesIO() as b:
+            try:
+                deployment.get_payload_by_id(item["payload_id"], b)
+            except ClientError as e:
+                if e.response["Error"]["Code"] == "404":
+                    logger.error(
+                        "Payload ID: '%s' was not found in S3",
+                        item["payload_id"],
+                    )
+                else:
+                    logger.error(
+                        "Error retrieving payload ID '%s': '%s'",
+                        item["payload_id"],
+                        e,
+                    )
+            b.seek(0)
+            payload = json.load(b)
+
+        payload["process"][0]["replace"] = True
```

Comment:
I would recommend moving this to an parameter/option, there might be cases where an operator is relying on cirrus to not reprocess items which did succeed.  The option I'd recommend the option default to False, until we see that you do always want this True.

```suggestion
        payload["process"][0]["replace"] = replace
```

Reply:
good point this actually dovetails with the thoughts I was having above.  Having it as a flag makes sense and keeps it from being confusing and that `get_payloads` defaults to just that, getting payloads